nevergrad/functions/gym/gym_multi.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+import os
+import gym
+
+if os.name != "nt":
+    import gym_anm  # type: ignore  # pylint: disable=unused-import
+from nevergrad.parametrization import parameter
+from ..base import ExperimentFunction
+
+
+# Method for building a new list, for a future version of gym:
+#
+# gym_env_names = ["gym_anm:ANM6Easy-v0"]
+#
+# for e in gym.envs.registry.all():
+#     try:
+#         assert "Kelly" not in e.id
+#         env = gym.make(e.id)
+#         a1 = env.action_space.sample()
+#         a2 = env.action_space.sample()
+#         a3 = env.action_space.sample()
+#         a1 = a1 + a2 + a3
+#         if hasattr(a1, "size"):
+#             assert a1.size() < 15000
+#         gym_env_names.append(e.id)
+#     except:
+#         pass
+
+gym_env_names = [
+    "gym_anm:ANM6Easy-v0",
+    "Copy-v0",
+    "RepeatCopy-v0",
+    "ReversedAddition-v0",
+    "ReversedAddition3-v0",
+    "DuplicatedInput-v0",
+    "Reverse-v0",
+    "CartPole-v0",
+    "CartPole-v1",
+    "MountainCar-v0",
+    "Acrobot-v1",
+    "Blackjack-v0",
+    "FrozenLake-v0",
+    "FrozenLake8x8-v0",
+    "CliffWalking-v0",
+    "NChain-v0",
+    "Roulette-v0",
+    "Taxi-v3",
+    "CubeCrash-v0",
+    "CubeCrashSparse-v0",
+    "CubeCrashScreenBecomesBlack-v0",
+    "MemorizeDigits-v0",
+]
+
+
+class GymMulti(ExperimentFunction):
+    def __init__(
+        self, name: str = "gym_anm:ANM6Easy-v0", control: str = "conformant", neural_factor: int = 2
+    ) -> None:
+        env = gym.make(name)
+        self.name = name + "__" + control + "__" + str(neural_factor)
```

Comment:
why 2 underscores?

Reply:
Because there is _ inside the control, so I find that a better parsing.

Reply:
weird but why not^^ other options would be to use commas instead for instance, but whaver is simpler.
By the way, this is shorter:
`"__".join([name, control, str(neural_factor)])`